src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,70 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+            => token.Parent is AccessorListSyntax accessorList &&
+                All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+            => node?.Parent is AccessorListSyntax accessorList &&
+                All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool HasInitializer(BasePropertyDeclarationSyntax basePropertyDeclaration)
+            => basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null;
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxToken token)
+            => token.Parent is AccessorListSyntax accessorList &&
+                accessorList.Parent is BasePropertyDeclarationSyntax basePropertyDeclarationSyntax &&
+                HasInitializer(basePropertyDeclarationSyntax);
+
+        private static int LineBreaksBeforeOpenBrace(SyntaxToken currentToken, SyntaxToken nextToken)
+        {
+            if (nextToken.Parent.IsKind(SyntaxKind.Interpolation) ||
+                nextToken.Parent is InitializerExpressionSyntax)
+            {
+                return 0;
+            }
+            else if (IsAutoAccessorList(nextToken))
+            {
+                // currentToken is the property name, nextToken is the open brace
+                return 0;
+            }
+            else
+            {
+                return 1;
+            }
+        }
+
+        private static int LineBreaksBeforeCloseBrace(SyntaxToken currentToken, SyntaxToken nextToken)
+        {
+            if (nextToken.Parent.IsKind(SyntaxKind.Interpolation) ||
+                nextToken.Parent is InitializerExpressionSyntax)
+            {
+                return 0;
+            }
+            else
+            {
+                return 1;
```

Comment:
>return 1; [](start = 16, length = 9)

I am curious why we don't need a special handling for a close brace. Could you elaborate please?

Reply:
At the close brace time, the formatting has already been done (either inline or expanded on multiple lines, indents, etc.)
But we still have to care about the initializer to avoid the two crlf that inserted in the current implementation